extension/src/background/helpers/account.ts

```diff
@@ -34,8 +37,17 @@ export const addAccountName = ({
   localStorage.setItem(ACCOUNT_NAME_LIST_ID, encodedaccountNameList);
 };
 
-export const getIsTestnet = () =>
-  JSON.parse(localStorage.getItem(IS_TESTNET_ID) || "false");
+export const getIsMainnet = () => {
+  const networkDetails = getNetworkDetails();
+
+  return isMainnet(networkDetails);
+};
+
+export const getIsTestnet = () => {
+  const networkDetails = getNetworkDetails();
+
+  return isTestnet(networkDetails);
+};
```

Comment:
Since `isTestnet` is no longer a binary choice, we need to add some new helpers to determine if a network is Mainnet or Testnet